package.json

```diff
@@ -33,7 +33,7 @@
     "ethers": "^4.0.39"
```

Comment:
swap is currently on 3.3.5. not sure if that's necessary for maker-kit but wanted to point that out.

Reply:
i thought i updated that.